File: myapp/parent/Step30_Lambda.cs
using sfn_ut.sfn.lambda;

namespace sfn_ut.myapp.parent;

public class Step30_Lambda : SfnLambda<ParentPayload> {
  public override ParentPayload Execute(ParentPayload payload) {
    payload.HappyPathData += " > 30";
    return payload;
  }
}

File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/parent/Step30_Lambda.java
package com.sfnut.myapp.parent;

import com.sfnut.sfn.lambda.SfnLambda;

public class Step30_Lambda extends SfnLambda<ParentPayload> {
    @Override
    public ParentPayload execute(ParentPayload payload) {
        payload.setHappyPathData(payload.getHappyPathData() + " > 30");
        return payload;
    }
}
